src/pipx/package_specifier.py

```diff
@@ -188,6 +188,23 @@ def parse_specifier_for_upgrade(package_spec: str) -> str:
     return package_or_url
 
 
+def get_extras(package_spec: str) -> Set[Any]:
```

Comment:
Aren’t extras supposed to only be strings? `packaging` declares `Requirement.extras` as `Set[str]`.

Reply:
Yes good catch.  Changed.